irohad/consensus/yac/impl/yac.cpp

```diff
@@ -24,10 +24,9 @@ namespace iroha {
     namespace yac {
 
       template <typename T>
-      std::string cryptoError(const model::Peer &peer, const T &votes) {
-        std::string result = "Crypto verification failed for message from ";
-        result += peer.address;
-        result += ". \nVotes: ";
+      std::string cryptoError(const T &votes) {
+        std::string result =
+            "Crypto verification failed for message.\n Votes: ";
         result += logger::to_string(votes, [](const auto &vote) {
           std::string result = "(Public key: ";
           result += vote.signature.pubkey.to_hexstring();
```

Comment:
Do not use concatenation in strings, use stringstream:
```
std::stringstream ss;
ss << "Msg1" << "msg2" << number << 1.337 << std::endl;
```

In this case you do not have to deal with types, and code looks cleaner.